### PR TITLE
ZK-2459: Add BindUtils postNotifyChange method accepting multiple pro…

### PR DESCRIPTION
**Pull request by charlesqqq**
…perty names

### CODE REVIEW

File: zktest/test/java/org/zkoss/zktest/zats/test2/F85_ZK_2459Test.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Copyright (C) 2018 Potix Corporation. All Rights Reserved.
+*/
+package org.zkoss.zktest.zats.test2;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import org.zkoss.zktest.zats.WebDriverTestCase;
+
+public class F85_ZK_2459Test extends WebDriverTestCase {
```
Comment (scribetw) [inline zktest/test/java/org/zkoss/zktest/zats/test2/F85_ZK_2459Test.java]: Seems it can be written in ZATS (No browser involved)

**Comment by scribetw** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io/reviews/zkoss/zk/1126)*

**Comment by wenninghsu** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I could not rebase F85-ZK-2459 with master. There are conflicts.

**Comment by wenninghsu** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I could not rebase F85-ZK-2459 with master. There are conflicts.

**Pull request merged**